### PR TITLE
fix: shadow resolution on Ultra is now 4096

### LINKED ISSUE #803: Shadow resolution does not change for Ultra preset
Ultra preset should have the best available graphics. It's not changing the shadow resolution. It should be : 4096

### PR DESCRIPTION
**Pull request by Kinerius**
Fixes #803

How to test:
- Run the game
- Set graphics preset to Ultra
- Shadow resolution should be 4096

### CODE REVIEW

**Review by BrianAmadori — APPROVED** [review]
🚀

**Comment by BrianAmadori** [comment]
For a change this small don't expect a second review btw, merge right away when you feel like it.

**Review by pbosio — APPROVED** [review]

**Review by kuruk-mm — APPROVED** [review]

**Pull request merged**
